info/firmware.info.xml

```diff
@@ -120,6 +120,13 @@
         </firmware>
     </board>
 
+    <board type="foc-special">
+        <firmware>
+            <file>../CAN/2foc/2foc-special.hex</file>
+            <version major='3' minor='3' build='26'/>
```

Comment:
just a comment: probably it is better to use  a build version that is different from the standard 2foc so that we can understand what version is deployed on the robot.

also because ... (if i remember correctly, so pls check) if it is used the --update param then the programming is skipped if a board already has the same application version.

Reply:
ciao @marcoaccame ! It's my fault! the rev is 3.3.27; I forgot to update in in the PR after the tests on the robot 👍🏼 
I'm going to commit the fix

Thank you!

cc @valegagge